src/requirements/common/GuildSelect.tsx

```diff
@@ -33,47 +40,32 @@ const GuildSelect = ({ baseFieldPath }) => {
 
   const { data: guildOptions, isValidating: isGuildsLoading } =
     useGuilds(debouncedSearchValue)
+  const { isLoading, ...currentGuild } = useGuild()
 
-  const guildId = useWatch({ name: `${baseFieldPath}.data.guildId` })
-
-  const { isLoading, ...fetchedGuild } = useGuild(
```

Comment:
I think we need this logic so users can search for hidden guilds too ;)

Reply:
Uhh true, i put it back